webapp/packages/plugin-authentication-administration/src/Administration/Users/Teams/GrantedConnections/GrantedConnectionsTabService.ts

```diff
@@ -95,15 +95,38 @@ export class GrantedConnectionsTabService extends Bootstrap {
       return;
     }
 
+    const { connectionsToRevoke, connectionsToGrant } = this.getConnectionsDifferences(state);
+
```

Comment:
`grantInfo` is not used as expected - it's the latest state of granted connections, but in `getConnectionsDifferences` you using `state.initialGrantedSubjects` instead, please use `grantInfo` (you can pass it as a second argument)